### DISCUSSION TITLE
VS Studio Version is generated on build, but on pack it generates 1.0.0

### DISCUSSION THREAD

**Discussion by Speed258** [devlooped/GitInfo#210] [Q&A] [answered]
I want to inherit version to nuget package, because file version is generated valid one, but for nuget it generates 1.0.0

project.csproj

To show code I removed < symbol
`Project Sdk="Microsoft.NET.Sdk">

  PropertyGroup>
	TargetFramework>net5.0</TargetFramework>
	GenerateAssemblyInfo>false</GenerateAssemblyInfo>
  /PropertyGroup>
	PropertyGroup>
		PackageId>test</PackageId>
		Authors>test</Authors>
		Company>test</Company>
		PackageDescription>test</PackageDescription>
		RepositoryUrl>https://github.com/test/test/</RepositoryUrl>
		GeneratePackageOnBuild>false</GeneratePackageOnBuild>
		RepositoryType>git</RepositoryType>
	/PropertyGroup>
	ItemGroup>
	  PackageReference Include="GitInfo" Version="2.2.0" />
	/ItemGroup>
/Project>`

Assemblyinfo.cs
`using System.Reflection;
using System.Runtime.InteropServices;

// General Information about an assembly is controlled through the following 
// set of attributes. Change these attribute values to modify the information
// associated with an assembly.
[assembly: AssemblyTitle("test")]
[assembly: AssemblyDescription("")]
[assembly: AssemblyConfiguration("")]
[assembly: AssemblyCompany("test")]
[assembly: AssemblyProduct("test")]
[assembly: AssemblyCopyright("Copyright © test 2022")]
[assembly: AssemblyTrademark("")]
[assembly: AssemblyCulture("")]

// Setting ComVisible to false makes the types in this assembly not visible 
// to COM components.  If you need to access a type in this assembly from 
// COM, set the ComVisible attribute to true on that type.
[assembly: ComVisible(false)]

// The following GUID is for the ID of the typelib if this project is exposed to COM
[assembly: Guid("99bd8b6c-33a1-4d65-bcb0-dd04be541b6b")]

// Version information for an assembly consists of the following four values:
//
//      Major Version
//      Minor Version 
//      Build Number
//      Revision
//
[assembly: AssemblyVersion(ThisAssembly.Git.BaseVersion.Major + "." + ThisAssembly.Git.BaseVersion.Minor + "." + ThisAssembly.Git.BaseVersion.Patch)]
[assembly: AssemblyFileVersion(ThisAssembly.Git.SemVer.Major + "." + ThisAssembly.Git.SemVer.Minor + "." + ThisAssembly.Git.SemVer.Patch)]
[assembly: AssemblyInformationalVersion(
    ThisAssembly.Git.SemVer.Major + "." + ThisAssembly.Git.SemVer.Minor + "." + ThisAssembly.Git.SemVer.Patch)]`

**Comment by kzu** [comment, marked as answer]
Documented at https://github.com/devlooped/GitInfo/pull/211
